news/_posts/releases/2020-09-01-prestashop-1-7-7-0-beta2-release.md

```diff
@@ -0,0 +1,200 @@
+- [Category root displayed after filter](https://github.com/PrestaShop/PrestaShop/issues/20067)
+- [Order of js loaded from module has been modified](https://github.com/PrestaShop/PrestaShop/issues/19244)
+- [No access to employee's profile when not an admin](https://github.com/PrestaShop/PrestaShop/issues/19077)
+- [In BO Product page, ecotax tax excl is used instead of ecotax tax incl](https://github.com/PrestaShop/PrestaShop/issues/10026)
+
+
+**Front Office:**
+- [Add the possibility to override the image used as cover](https://github.com/PrestaShop/PrestaShop/issues/19029) reported by [prestamodule](https://github.com/prestamodule), read the [documentation](https://devdocs.prestashop.com/1.7/faq/product/#product-cover)
+- [Fixed a bug where the voucher was not displayed if linked to a customer](https://github.com/PrestaShop/PrestaShop/issues/19442) reported by [clotaire202](https://github.com/clotaire202)
+- [Fixed the impossibility to select the language from the FO after importing a localization pack](https://github.com/PrestaShop/PrestaShop/issues/19893)
+- [Detail discount is not good in multicurrency context](https://github.com/PrestaShop/PrestaShop/issues/18900)
+
+**Core:**
+- [Fixed Doctrine Entities from Core that are inaccessible in front office](https://github.com/PrestaShop/PrestaShop/issues/19015)
+- [AdminModules : Call to a member function isUsingNewTranslationSystem](https://github.com/PrestaShop/PrestaShop/issues/19447)
+- [Fixed a syntax error in a SQL query](https://github.com/PrestaShop/PrestaShop/issues/19273)
+- [Fixed a hook missing/misspelt in DB: actionAuthenticationBefore](https://github.com/PrestaShop/PrestaShop/issues/18945)
+- [Fixed Cache Problem in Language & Currency](https://github.com/PrestaShop/PrestaShop/pull/20398)
+- [Computing precision: set minimum computing precision value to zero](https://github.com/PrestaShop/PrestaShop/pull/20137)
+- [Auto create symfony roles on tab register](https://github.com/PrestaShop/PrestaShop/issues/19857)
+- [Add trace information to Command/Query debugger](https://github.com/PrestaShop/PrestaShop/pull/19854)
+- [Allow modules to disable routing prefix with specific parameter](https://github.com/PrestaShop/PrestaShop/pull/19782)
+- [Update decimal library](https://github.com/PrestaShop/PrestaShop/pull/19748)
+- [Update modules (contactform/ps_imageslider/ps_sharebuttons/statspersonalinfos)](https://github.com/PrestaShop/PrestaShop/pull/19556)
+- [Introduce CompilerPass to perform ControllerAwareTrait injection](https://github.com/PrestaShop/PrestaShop/pull/18811)
+- [Add tag controller.service_arguments to PrestaShop controllers](https://github.com/PrestaShop/PrestaShop/pull/18845)
+
+**Installation:**
+- [Fix session tables charset](https://github.com/PrestaShop/PrestaShop/pull/20122)
+- [Error when installing without debug mode disabled](https://github.com/PrestaShop/PrestaShop/issues/19754)
+
+**Circuit breaker:**
```

Comment:
I don't think people need to know the specifics about circuit breaker. To them it's just a reliablity
```suggestion
**Reliability:**
```